code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -284,33 +284,37 @@
 	return ..() | update_flags
 
 /datum/reagent/acid/reaction_mob(mob/living/M, method = REAGENT_TOUCH, volume)
-	if(ishuman(M) && !isgrey(M))
-		var/mob/living/carbon/human/H = M
-		if(method == REAGENT_TOUCH)
-			if(volume > 25)
-				if(H.wear_mask)
-					to_chat(H, "<span class='danger'>Your [H.wear_mask] protects you from the acid!</span>")
-					return
-
-				if(H.head)
-					to_chat(H, "<span class='danger'>Your [H.wear_mask] protects you from the acid!</span>")
-					return
-
-				if(prob(75))
-					H.take_organ_damage(5, 10)
-					H.emote("scream")
-					var/obj/item/organ/external/affecting = H.get_organ("head")
-					if(istype(affecting))
-						affecting.disfigure()
-				else
-					H.take_organ_damage(5, 10)
-			else
-				H.take_organ_damage(5, 10)
+	if(!ishuman(M) || isgrey(M))
+		return
+
+	var/mob/living/carbon/human/H = M
+	if(!(method == REAGENT_TOUCH))
+		to_chat(H, "<span class='warning'>The greenish acidic substance stings[volume < 10 ? " you, but isn't concentrated enough to harm you" : null]!</span>")
+		if(volume >= 10)
+			H.adjustFireLoss(clamp((volume * 2), 4, 20))
+			H.emote("scream")
+		return
+
+	if(volume < 25) // Need at least 10 units to do a little bit of damage
+		if(volume > 10)
+			H.take_organ_damage(5, 10)
 		else
-			to_chat(H, "<span class='warning'>The greenish acidic substance stings[volume < 10 ? " you, but isn't concentrated enough to harm you" : null]!</span>")
-			if(volume >= 10)
-				H.adjustFireLoss(min(max(4, (volume - 10) * 2), 20))
-				H.emote("scream")
+			H.adjustFireLoss(clamp(volume, 1, 5))
+		return
+
+	if(H.wear_mask)
+		to_chat(H, "<span class='danger'>Your [H.wear_mask] protects you from the acid!</span>")
+		return
+	if(H.head)
+		to_chat(H, "<span class='danger'>Your [H.wear_mask] protects you from the acid!</span>")
+		return
```

Comment:
I'm guessing this is what you meant to do here?
```suggestion
	if(H.head)
		to_chat(H, "<span class='danger'>Your [H.head] protects you from the acid!</span>")
		return
```

Reply:
Yep! Slight copy paste error